projects/core/src/global-message/http-interceptors/handlers/bad-request/bad-request.handler.ts

```diff
@@ -39,7 +39,9 @@ export class BadRequestHandler extends HttpErrorHandler {
     ) {
       this.globalMessageService.add(
         {
-          key: 'httpHandlers.badRequestPleaseLoginAgain',
+          key: `httpHandlers.badRequest.${this.createErrorTranslationKey(
```

Comment:
Could be have a condition to revert to the old message in case `error_description` does not exist? (I don't know if we need to worry about this case.)

We can have a `getBadRequestTranslationKey` method return the new key if the `error_description` exists or return `httpHandlers.badRequestPleaseLoginAgain`. We can also handle the breaking change in this method if this is going into 5.2.

Reply:
I added a statement with a feature flag check and if a parameter is not a string. Is it what you meant?

Reply:
And yes, hopefully, it should go into 5.2